sdk/monitor/monitor-opentelemetry-exporter/src/export/statsbeat/networkStatsbeatMetrics.ts

```diff
@@ -341,7 +341,7 @@ export class NetworkStatsbeatMetrics extends StatsbeatMetrics {
   private getShortHost(originalHost: string): string {
     let shortHost = originalHost;
     try {
-      const hostRegex = new RegExp(/^https?:\/\/(?:www\.)?([^\/.-]+)/);
```

Comment:
Do you know the reasoning for the change on this regex?

Reply:
The error was 
> Unnecessary escape character: \/.